psi4/src/psi4/libtrans/integraltransform.cc

```diff
@@ -44,240 +44,221 @@
-                                     FrozenOrbitals frozenOrbitals,
-                                     bool init):
-    initialized_(false),
-    psio_(_default_psio_lib_),
-    transformationType_(transformationType),
-    uniqueSpaces_(spaces),
-    moOrdering_(moOrdering),
-    outputType_(outputType),
-    frozenOrbitals_(frozenOrbitals),
-    alreadyPresorted_(false),
-    dpdIntFile_(PSIF_LIBTRANS_DPD),
-    aHtIntFile_(PSIF_LIBTRANS_A_HT),
-    bHtIntFile_(PSIF_LIBTRANS_B_HT),
-    nirreps_(0),
-    nmo_(0),
-    nso_(0),
-    nTriSo_(0),
-    nTriMo_(0),
-    nfzc_(0),
-    nfzv_(0),
-    spaces_(0),
-    labels_(0),
-    tolerance_(1.0E-16),
-    memory_(250 * 1024 * 1024),
-    moIntFileAA_(0),
-    moIntFileAB_(0),
-    moIntFileBB_(0),
-    myDPDNum_(1),
-    print_(1),
-    zeros_(0),
-    sopi_(0),
-    sosym_(0),
-    mosym_(0),
-    mopi_(0),
-    clsdpi_(0),
-    openpi_(0),
-    frzcpi_(0),
-    frzvpi_(0),
-    aQT_(0),
-    bQT_(0),
-    cacheFiles_(0),
-    cacheList_(0),
-    aCorrToPitzer_(0),
-    bCorrToPitzer_(0),
-    keepIwlSoInts_(false),
-    keepIwlMoTpdm_(true),
-    keepDpdSoInts_(false),
-    keepDpdMoTpdm_(true),
-    keepHtInts_(true),
-    keepHtTpdm_(true),
-    tpdmAlreadyPresorted_(false),
-    soIntTEIFile_(PSIF_SO_TEI)
-{
+IntegralTransform::IntegralTransform(SharedMatrix H, SharedMatrix c, SharedMatrix i, SharedMatrix a, SharedMatrix v,
+                                     SpaceVec spaces, TransformationType transformationType, OutputType outputType,
+                                     MOOrdering moOrdering, FrozenOrbitals frozenOrbitals, bool init)
+    : initialized_(false),
+      psio_(_default_psio_lib_),
+      transformationType_(transformationType),
+      uniqueSpaces_(spaces),
+      moOrdering_(moOrdering),
+      outputType_(outputType),
+      frozenOrbitals_(frozenOrbitals),
+      alreadyPresorted_(false),
+      dpdIntFile_(PSIF_LIBTRANS_DPD),
+      aHtIntFile_(PSIF_LIBTRANS_A_HT),
+      bHtIntFile_(PSIF_LIBTRANS_B_HT),
+      nirreps_(0),
+      nmo_(0),
+      nso_(0),
+      nTriSo_(0),
+      nTriMo_(0),
+      nfzc_(0),
+      nfzv_(0),
+      spaces_(0),
+      labels_(0),
+      tolerance_(1.0E-16),
+      memory_(250 * 1024 * 1024),
+      moIntFileAA_(0),
+      moIntFileAB_(0),
+      moIntFileBB_(0),
+      myDPDNum_(1),
+      print_(1),
+      zeros_(0),
+      sopi_(0),
+      sosym_(0),
+      mosym_(0),
+      mopi_(0),
+      clsdpi_(0),
+      openpi_(0),
+      frzcpi_(0),
+      frzvpi_(0),
+      aQT_(0),
+      bQT_(0),
+      cacheFiles_(0),
+      cacheList_(0),
+      aCorrToPitzer_(0),
+      bCorrToPitzer_(0),
+      keepIwlSoInts_(false),
+      keepIwlMoTpdm_(true),
+      keepDpdSoInts_(false),
+      keepDpdMoTpdm_(true),
+      keepHtInts_(true),
+      keepHtTpdm_(true),
+      tpdmAlreadyPresorted_(false),
+      soIntTEIFile_(PSIF_SO_TEI) {
     memory_ = Process::environment.get_memory();
 
-    nirreps_  = c->nirrep();
```

Comment:
Just be aware you are removing a section of formatting that was hand built to improve readability here.